app/src/main/res/layout/fragment_fever_temperature_spot.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto" >
+    <data>
+        <variable
+            name="vm"
+            type="org.coepi.android.ui.symptoms.fever.FeverTemperatureSpotViewModel" />
+    </data>
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:background="@drawable/ic_geometric_light_background"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+        <androidx.appcompat.widget.Toolbar
+            android:id="@+id/toolbar"
+            android:layout_width="match_parent"
+            android:layout_height="?attr/actionBarSize"
+            android:background="@color/colorAccent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:navigationIcon="@drawable/ic_arrow_back_white_24dp"
+            app:title="@string/symptom_report_fever_title"
+            app:titleTextColor="@android:color/white">
+        </androidx.appcompat.widget.Toolbar>
+
+        <TextView
+            android:id="@+id/headline"
+            android:name="androidx.navigation.fragment.NavHostFragment"
+            style="@style/Text.Headline"
+            android:textStyle="bold"
+            android:textColor="@color/white"
+            android:background="@color/colorPrimary"
+            android:layout_width="0dp"
+            android:layout_height="109dp"
+            android:paddingStart="32dp"
+            android:gravity="left|center_vertical"
+            android:text="@string/symptom_report_fever_temperature_spot_headline"
+            android:textSize="@dimen/text_size_headline"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/toolbar" />
+        <LinearLayout
+            android:id="@+id/options"
+            android:orientation="vertical"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="44dp"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/headline">
+        <LinearLayout
```

Comment:
There's a lot of nesting. It should be possible to show the 4 buttons directly in a constraint layout.

Reply:
Reduced nesting and put everything into the constraint layout.